appsettings/src/tree/AppSettingsTreeItem.ts

```diff
@@ -74,6 +74,19 @@ export class AppSettingsTreeItem extends AzExtParentTreeItem {
         }
     }
 
+    static async createAppSettingsTreeItem(context: IActionContext, parent: AzExtParentTreeItem, clientProvider: AppSettingsClientProvider, extensionPrefix: string, options?: AppSettingsTreeItemOptions): Promise<AppSettingsTreeItem> {
+        const client = await clientProvider.createClient(context);
+        const appSettings = await client.listApplicationSettings();
+        if (appSettings.properties) {
+            for (const [key, value] of Object.entries(appSettings.properties)) {
+                if (isSettingConvertible(key, value)) {
+                    options?.contextValuesToAdd?.push('convert');
+                }
+            }
+        }
+        return new AppSettingsTreeItem(parent, clientProvider, extensionPrefix, options);
```

Comment:
I'd also add a `refreshImpl` implementation that does this logic and then call `this.refreshImpl` in the static create function.  That way if a user refreshes the node, it will update the settings.